src/find_library.cpp

```diff
@@ -21,10 +21,13 @@
 #include <fstream>
 #include <sstream>
 #include <string>
+#include <vector>
```

Comment:
Do you need to remove `#include <list>` which is above this line?

Reply:
removed https://github.com/ros2/rcpputils/pull/54/commits/1b24ab54c1e36b3bc57763cdb4755d1bf4dddb03